disnake/raw_models.py

```diff
@@ -147,6 +150,62 @@ def __init__(self, data: MessageUpdateEvent) -> None:
             self.guild_id: Optional[int] = None
 
 
+PollEventType = Literal["POLL_VOTE_ADD", "POLL_VOTE_REMOVE"]
+
+
+class RawMessagePollVoteActionEvent(_RawReprMixin):
+    """Represents the event payload for :func:`on_raw_message_poll_vote_add` and
+    :func:`on_raw_message_poll_vote_remove` events.
+
+    .. versionadded:: 2.10
+
+    Attributes
+    ----------
+    message_id: :class:`int`
+        The message ID that got or lost a vote.
+    user_id: :class:`int`
+        The user ID who added the vote or whose vote was removed.
+    member: Optional[:class:`Member`]
```

Comment:
Although this is not consistent across all raw models, changing this to `cached_member` would be more descriptive.